internal/github/issue.go

```diff
@@ -102,3 +103,12 @@ func (ih *IssueHelperImpl) ListAllOpen(ctx context.Context, includePRs bool) ([]
 
 	return i, nil
 }
+
+func (ih *IssueHelperImpl) Assign(ctx context.Context, issue *github.Issue, user *github.User) error {
```

Comment:
I think this function can receive just a `string` for the user login. That should simplify other functions.

Reply:
I though about it as well but decided to keep the user in purpose in case we will want some addition user info in the future.

Reply:
[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) 🙂

Reply:
Fixed.